ReactiveCocoaTests/Swift/PropertySpec.swift

```diff
@@ -101,10 +168,10 @@ class PropertySpec: QuickSpec {
 					property.producer.startWithNext { x in value = x }
 				}
 
-				expect(value).to(equal(0))
+				expect(value) == 0
```

Comment:
Hmm. I have overlooked this which is not a new test case introduced in this PR, and applied the new Nimble matchers to it anyway as part of the requested change from @NachoSoto. Please tell in case it has to be reverted.